doc/jemalloc.xml.in

```diff
@@ -2156,6 +2190,42 @@ struct extent_hooks_s {
         </para></listitem>
       </varlistentry>
 
+      <varlistentry id="stats.background_thread.num_threads">
+        <term>
+          <mallctl>stats.background_thread.num_threads</mallctl>
+          (<type>size_t</type>)
+          <literal>r-</literal>
+          [<option>--enable-stats</option>]
+        </term>
+        <listitem><para> Number of <link linkend="background_thread">background
+        threads</link> running currently.  Return 0 if
```

Comment:
```s/Return /Returns /```, here and elsewhere.  That said, I don't think we need to document the 0 return case for any of these, since the meaning of 0 return is self-evident.